packages/core/lib/console-child.js

```diff
@@ -23,41 +24,71 @@ global.crypto = {
 const { network, config, url } = yargs(input[0]).argv;
 const detectedConfig = Config.detect({ network, config });
 
-function getConfiguredNetworkUrl(customConfig) {
+function getConfiguredNetworkUrl(customConfig, isDashboardNetwork) {
+  const defaultPort = isDashboardNetwork
+    ? managedDashboardDefaultPort
+    : managedGanacheDefaultPort;
   const configuredNetworkOptions = {
-    host: customConfig.host || managedGanacheDefaultHost,
-    port: customConfig.port || managedGanacheDefaultPort
+    host: customConfig.host || defaultHost,
+    port: customConfig.port || defaultPort
   };
-  return `http://${configuredNetworkOptions.host}:${configuredNetworkOptions.port}/`;
+  const urlSuffix = isDashboardNetwork ? "/rpc" : "";
+  return `http://${configuredNetworkOptions.host}:${configuredNetworkOptions.port}${urlSuffix}`;
 }
 
 let configuredNetwork;
 
-if (url) {
+const configNetworkWithProvider =
+  detectedConfig.networks[network] && detectedConfig.networks[network].provider;
+
+if (configNetworkWithProvider) {
+  // Use "provider" specified in the network config
```

Comment:
Should this be updated to indicate other configured network properties are used as well?

Reply:
Yes, this comment needs to be updated I guess, to indicate spreading out of the other specified network properties. Thanks for this @cds-amal.